kilkari/src/main/java/org/motechproject/nms/kilkari/domain/Subscription.java

```diff
@@ -106,7 +118,7 @@ public void setNeedsWelcomeMessage(boolean needsWelcomeMessage) {
     }
 
     /**
-     * Helper method to be called by the OBD process when selecting a message to play for a subscription
+     * Helper method to be called by the OBD processCallDetailRecord when selecting a message to play for a subscription
```

Comment:
Unrelated to your current change, but this seems like a lot of business logic to exist in a domain object that we would consider to be a data container, right? Should be move this to a helper method? 

I'm torn on this this design pattern http://java.dzone.com/articles/business-logic-domain-objects